torchinductor/config.py

```diff
@@ -101,3 +103,33 @@ class triton:
     # should we stop a fusion to allow better tiling?
     tiling_prevents_pointwise_fusion = True
     tiling_prevents_reduction_fusion = True
+
+
+# create a directory containing lots of debug information
+class trace:
+    # master switch for all debugging flags below
+    enabled = os.environ.get("TORCHINDUCTOR_TRACE", "0") == "1"
+
+    # Save python logger call >=logging.DEBUG
+    debug_log = True
+
+    # Save python logger call >=logging.INFO
+    info_log = False
```

Comment:
Maybe just `config.trace.save_log_level`?

Reply:
I actually find it useful to save both.  The debug level log can get too noisy, but it is nice to have it to reference if I want more details.